VaticanBAV/et/BAVet196.xml

```diff
@@ -163,6 +164,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                           </layoutDesc>
                         </objectDesc>
 
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1v" />
+                              <desc type="Supplication">
+                                <title>ʾAsmāt Prayer</title>
```

Comment:
there is a generic ID for ʾAsmāt-texts that could be added in a `@ref`

Reply:
How has this been resolved? It would also be possible to use `@type="Asmat"` and then mark up the quotated text with `<seg type="supplication">`